src/api/authed/intake/neighborhood-finder.js

```diff
@@ -0,0 +1,45 @@
+const {
+  updateRequestByCode,
+  findRequestByCode
+} = require("../../../airtable.js");
+
+const NEIGHBORHOOD_AREA_UNAVAILABLE_OPTION = "Other - not Crown Heights";
```

Comment:
Wish there was a better way so i didn't have to hardcode airtable values. Please let me know if there is!

Reply:
We've been talking about centralizing airtable constants, but is still just tech debt. we can put it off for now